LongitudinalDifferenceInDifferencesPy/didDesigner.py

```diff
@@ -254,6 +254,30 @@ def visualizeDifferenceInDifferences(self,df,yLabel,title):
         plt.grid(True)
         plt.show()
 
+    def printTreatmentMatrixPoints(self, df):
+        '''
+        Prints out KPIs of pre-treatment and post-treatment of the comparison and treated groups.
+        '''
+        treatmentMatrix = self._calculate_aggregate_treatment_matrix(df, aggregate=np.mean)
+        kpiTreat0Post0_mean, kpiTreat0Post1_mean, kpiTreat1Post0_mean, kpiTreat1Post1_mean = treatmentMatrix
+        print('pre-treatment mean KPI of comparison group: ' + kpiTreat0Post0_mean + '\npost-treatment mean KPI of comparison group: ' + kpiTreat0Post1_mean + '\npre-treatment mean KPI of treated group: ' + kpiTreat1Post0_mean + '\npost-treatment mean KPI of treated group: ' + kpikpiTreat1Post1_mean)
+
+    def calculatePercentImprovement(self, df):
+        '''
+        Calculates and prints out the percent improvement of the treated group in comparison to comparison group.
+        '''
+        treatmentMatrix = self._calculate_aggregate_treatment_matrix(df, aggregate=np.mean)
+        kpiTreat0Post0_mean, kpiTreat0Post1_mean, kpiTreat1Post0_mean, kpiTreat1Post1_mean = treatmentMatrix
+        percentImprovement = ((kpiTreat1Post1_mean - (kpiTreat1Post0_mean + (kpiTreat0Post1_mean - kpiTreat0Post0_mean)))/(kpiTreat1Post0_mean + (kpiTreat0Post1_mean - kpiTreat0Post0_mean)))*100
```

Comment:
can you make the second term in the numerator a variable? you are reusing it in the denom. Don't repeat yourself!